feeder/ethereum/README.md

```diff
@@ -0,0 +1,24 @@
+# Ethereum Witness Feeder
+
+## Explanation
+
+The Ethereum Witness Feeder is a script used to periodically "feed" CT log data provided by public APIs to an Ethereum smart contract.
+
+See the [Ethereum Witness README](https://github.com/google/trillian-examples/blob/master/witness/ethereum/README.md) for additional context.
+
+## To run
+
+- cd into this directory (i.e. `feeder/ethereum`)
+- make sure your local environment matches the `.env.example` file in this directory, either via setting a local `.env` file in this directory, or manually passing in the env vars
```

Comment:
Not sure I understand what setting local env file means / how to do it. Do you mean something like `source .env.example` command? Maybe give an example of command that should do the trick.